snooty/util.py

```diff
@@ -192,13 +192,21 @@ def __len__(self) -> int:
         return sum(len(w) for w in self.directories.values())
 
 
+# Should this really default to flag-like behavior? Seems iffy at best
```

Comment:
I think changing this behavior could have pretty wide-ranging implications across repositories! There are lots of different boolean options and we want to preserve the usage expected by docutils, and yield the same output. Did you change this because you were running in to problems?

Reply:
With regards to the function, I am indeed running into issues where `:copyable: false` and `:copyable: False` both output `True`. I've temporarily edited this function for the commit -- we should definitely chat about the line `docutils.parsers.rst.directives.choice(argument, ("true", "false"))`.